packages/manager/modules/emailpro/src/redirection/redirection.routing.js

```diff
@@ -0,0 +1,12 @@
+import clone from 'lodash/clone';
+import template from './emailpro-redirection.html';
+
+const state = {
+  url: '/redirection',
+  template,
+  controller: 'EmailMXPlanEmailRedirectionCtrl',
+};
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('mxplan.dashboard.redirection', clone(state));
```

Comment:
You need to clone it here, even if you declare it just once ?

Reply:
Copy/Paste and forgot to clean it up